packages/data-context/test/unit/actions/ProjectActions.spec.ts

```diff
@@ -3,6 +3,7 @@ import { ProjectActions } from '../../../src/actions/ProjectActions'
 import { createTestDataContext } from '../helper'
 import { expect } from 'chai'
 import sinon from 'sinon'
+import path from 'path'
```

Comment:
I wonder if we need `CodegenActions` soon... WDYT?

Reply:
I like the idea of splitting these out, just to keep things organized

Reply:
@astone123 thoughts? Not sure how this fits into the timeline, happy to help out if you think this is a good idea -- as much as I want to say "we can do this later", I really doubt we actually will. Might be good to make this refactor now - I don't think it's a lot of work.

Reply:
Created `GodegenActions` in https://github.com/cypress-io/cypress/pull/24881/commits/9449d66fbfadd25c70dad78bd243f6fde64b999d